plone/app/portlets/portlets/actions.pt

```diff
@@ -4,15 +4,15 @@
       tal:omit-tag=""
       i18n:domain="plone">
 
-<section class="portlet portletActions" role="menu"
+<div class="portlet portletActions"
```

Comment:
Hm, when I try `menu` I get a warning from https://validator.w3.org/
"The menu element is not supported in all browsers. Please be sure to test, and consider using a polyfill."
I have changed them all to `asides` now and have rebased this pull request.  Maybe change some to menu another time...